src/main/java/me/totalfreedom/totalfreedommod/command/Command_spectator.java

```diff
@@ -7,7 +7,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.ADMIN, source = SourceType.BOTH)
-@CommandParameters(description = "Quickly change your own gamemode to spectator, or define someone's username to change theirs.", usage = "/<command> <[partialname]>", aliases = "gmsp")
+@CommandParameters(description = "Quickly change your own gamemode to spectator, or set another player's gamemode to spectator.", usage = "/<command> [player]", aliases = "gmsp")
```

Comment:
Any reason this doesn't also have a -a flag?

Reply:
Ops and MBs can't go into spectator so therefor it was never added since it wasn't needed